documentation/use-cases/Story2-UseCaseNarrative.md

```diff
@@ -0,0 +1,49 @@
+# STORY 2 - USE CASE NARRATIVE
+
+<b>Use case</b>: Record daily lecture topics  
+
+<b>Scope</b>:  /   
+
+<b>Level</b>: User-goal  
+
+<b>Intention in context</b>: The teacher wants to record lecture activities to to keep track of work done and inform parents.  
+
+<b>Primary actor</b>: Teacher.  
+
+<b>Secondary actor(s)</b>:  /  
+
+<b>Stakeholders' interests</b>:  
+	
+* Teacher: would like to easily and quickly record lecture activities, without having to think about the formatting of the text and with the possibility of modifying it if wrong. 
+* Parent: would like easily understand topics of the lecture.
+
+<b>Precondition</b>: 
+
+* There should be an authorized teacher in a valid open lecture
```

Comment:
Se il senso era autenticarsi, cambierei in "The teacher should have logged in"

Reply:
La penso come Gabriele, considerando che poi ci deve anche essere la lezione aperta direi "The teacher should be logged and a lecture has to be open"

Reply:
fatto